docs/spec/src/integration/spec/3-datastructs.md

```diff
@@ -42,38 +42,40 @@ Once an interpretation of the data has been chosen, one can convert between them
 
 ```solidity
 PolyCoeff --FFT--> PolyEval
-PolyCoef <--IFFT-- PolyEval
+PolyCoeff <--IFFT-- PolyEval
 ```
 
-EigenDA works differently from Ethereum  network interprets blobs as coefficients, only `PolyCoeff`s can be submitted as a `blob` to the Disperser. A `PolyEval` will thus need to be IFFT’d into a `PolyCoeff` before being dispersed. One benefit of interpreting the data as being evaluations is that it one can use point opening proofs to reveal a single field element (32 byte chunk) of data at a time. This is useful for interactive fraud proofs (e.g. see how [optimism fraud proves 4844 blobs](https://specs.optimism.io/fault-proof/index.html#type-5-global-eip-4844-point-evaluation-key)), but less so for validity proofs that zk rollups use.
+Whereas Ethereum treats 4844 blobs as evaluations of a polynomial, EigenDA instead interprets EigenDA blobs as coefficients of a polynomial. Thus, only `PolyCoeff`s can be submitted as a `blob` to the Disperser. Each rollup integration must thus decide whether to interpret their `encodedPayload`s as `PolyCoeff`, which can directly be dispersed, or as `PolyEval`, which will require IFFT’ing into a `PolyCoeff` before being dispersed. 
+
+Typically, optimistic rollups will interpret the data as being evaluations. This allows creating point opening proofs to reveal a single field element (32 byte chunk) at a time, which is needed for interactive fraud proofs (e.g. see how [optimism fraud proves 4844 blobs](https://specs.optimism.io/fault-proof/index.html#type-5-global-eip-4844-point-evaluation-key)). ZK rollups, on the flip side, don't require point opening proofs and thus can safely save up on the extra IFFT compute costs and instead interpret their data as coefficients directly.
```

Comment:
"...safely save ~~up~~ on the extra IFFT compute costs..."

Reply:
[ef87142](https://github.com/Layr-Labs/eigenda/pull/1439/commits/ef87142760d5df698336233d7972970076a4e6d5)